### DISCUSSION TITLE
Timeframe for AZ Cli 2.37.0 to be available on Azure DevOps Microsoft Hosted pipelines?

### DISCUSSION THREAD

**Discussion by pmatthews05** [actions/virtual-environments#5666] [Image configuration]
AZ Cli has recently updated to 2.37.0 and has breaking changes. Mainly because they have dropped Azure Windows Graph API and now using Microsoft Graph API instead. 

We have many pipelines that utilises az cli, and we are preparing the scripts when the Azure DevOps Microsoft Hosted gets upgraded to use the latest AZ Cli.

Are you able to give a timescale when you expect the latest Virtual Machines will be upgrade to host the latest Az Cli?

**Comment by miketimofeev** [comment]
Hi @pmatthews05!
What OS do you use for az cli tasks?

**Reply by pmatthews05** [reply]
Majority of our pipelines use "ubuntu-latest", although we do have some "windows-latest". 

Our daily pipelines run all on ubuntu-latest as far as I can tell.

**Reply by miketimofeev** [reply]
Then both will be updated with the new Azure Cli version in 24 hours.

**Reply by pmatthews05** [reply]
Just noticed that couple of my DevOps Microsoft Hosted pipelines are now using 2.37.0.
